variables.tf

```diff
@@ -6,21 +6,13 @@ variable "bucket_name" {
 
 variable "bucket_suffix" {
   default     = "default"
-  description = "Suffix to apply to the bucket (use `bucket_name` or `bucket_suffix`). When using `bucket_suffix`, the bucket name will be `[ACCOUNT_ID]-[REGION]-s3logging-[BUCKET_SUFFIX]."
+  description = "Suffix to apply to the bucket (use `bucket_name` or `bucket_suffix`). When using `bucket_suffix`, the bucket name will be `[account_id]-[region]-s3logging-[bucket_suffix]."
   type        = string
 }
 
 variable "lifecycle_rules" {
-  default = [{
-    id                            = "expire"
-    enabled                       = true
-    prefix                        = null
-    expiration                    = 2147483647
-    noncurrent_version_expiration = 365
-  }]
-
+  default     = []
```

Comment:
You're defaulting to no lifecycle rules now? Are you sure?

I'd kindof rather see this with no default so that the user has to explicitly pass an empty list if they want no rules, but I don't feel strongly enough to block on this.

Reply:
Yeah, this returns to the intended behavior - don't delete shit from people's buckets. Not setting at all and setting to 2^31 are the two available options AWS offers. All of your PR feedback points to setting to 2^31 as annoying, and I agree. I did make the basic example include a 1 year expiration for objects and versions though, which I think is where a statement about what you should do belongs.

Reply:
Fair enough.